src/controllers/objecting.entity.name.controller.ts

```diff
@@ -8,5 +11,16 @@ import { OBJECTIONS_COMPANY_NUMBER } from "../model/page.urls";
  * @param next
  */
 export const post = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
-  return res.redirect(OBJECTIONS_COMPANY_NUMBER)
+  const shareIdentityString: string = req.body.shareIdentity;
+  const fullNameValue: string = req.body.fullName;
+  const shareIdentityValue: boolean = (shareIdentityString === "yes") ? true :  false;
```

Comment:
intellij highlighted this line:
'(shareIdentityString === "yes") ? true : false' can be simplified to '(shareIdentityString === "yes")' 

I can't see shareIdentityString used elsewhere so it can be simplified to:
'(req.body.shareIdentity === "yes")'

Reply:
Simplified